certif/app/controllers/authenticated/team/invite.js

```diff
@@ -5,13 +5,17 @@ import { action } from '@ember/object';
 export default class AuthenticatedTeamInviteController extends Controller {
   @service router;
 
+  email = null;
+
   @action
   async createCertificationCenterInvitation(event) {
     event.preventDefault();
   }
 
   @action
-  updateEmail() {}
+  updateEmail(event) {
+    this.email = event.target?.value?.trim();
```

Comment:
Ne serait il pas interessant de dégager tout les espaces?
```suggestion
    this.email = event.target?.value?.replace(/\s/g, '')
```

Reply:
On a fait cette modification vers la fin avec un commit qui touche le front et le back avec cette regexp qui retire les espaces mais aussi les retours à la ligne `/[\s\r\n]/g`